jujugui/static/gui/src/app/components/profile/credential-list/_credential-list.scss

```diff
@@ -124,6 +124,25 @@
   .dropdown-menu__list-item-link {
     padding: 0 0 0.5rem;
   }
+
+  // XXX These override styles are required until the <PopUp> components is converted to Vanilla v1
+  .button-row {
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-gap: 1rem;
+
+    button {
+      width: 100%;
+    }
+
+    .button--destructive {
```

Comment:
Can you use p-button--negative?

Reply:
No, as `.button--destructive` is part of the <PopUp> component which is used elsewhere in the app in places that are not encapsulated in the `v1` namespace. Changing the markup of the button would fix it in Profile but break it everywhere else, hence the comment above this which is that these override styles are necessary until `<PopUp>` has been converted.